README.md

```diff
@@ -167,6 +167,18 @@ Type: `String` (optional)
 
 Brand Title is the brand of your report, e.g. Smoke Tests Report, Acceptance Test Report etc as per your need. If not passed, it will be displayed as "Cucumberjs Report"
 
+#### `timeInNanoseconds`
+Type: `Boolean`
+Default: `false`
+
+Set this to `true` if your version of cucumber outputs the test durations in nanoseconds.
+
+#### `mergeBackgroundSteps`
```

Comment:
Could you please explain what does it do?

Reply:
Ok, I just noticed that this is a special issue with cucumber-jvm. cucumber-js already does this, but jvm puts the background on the same level as the scenarios, so before every scenario there is a tab called background that just contains the single step. I can post screenshots on monday so it will become clear what I mean.

Reply:
Sounds good. Thanks, @newcat !

Reply:
Ok, so I wrote a little test setup with this feature:
```
Feature: Cucumber-JVM background problem

  Background:
    Given this step runs before every test
    And this step as well

  Scenario: A test with background
    Given it ran the background steps
    When I do nothing
    Then nothing should be done
```
Then it will output this:
So basically there is a background tab before every scenario tab. But I want the steps defined in the background tab to appear in the scenario tab and there should be no background tab at all. This can be achieved by setting the mergeBackgroundSteps option. It would then look like this:

Reply:
I wonder why Cucumber produces the JSON with the seperate Background step. Can you see a seperate Background object?

When we run the `https://github.com/gkushang/cucumber-html-reporter/blob/develop/test/features/happy/happy.feature#L9` which has background, it attaches the background step with the Scenario itself.


```
   {
            "arguments": [],
            "keyword": "When ",
            "line": 9,
            "name": "this feature runs with background",
            "match": {
              "location": "test/features/step_definitions/stepDefs.js:7"
            },
            "result": {
              "status": "passed",
              "duration": 1000000
            }
          },
```

<img width="570" alt="screen shot 2018-02-06 at 11 01 44 am" src="https://user-images.githubusercontent.com/3663389/35869496-2d486f30-0b2d-11e8-8430-d5a3aa815297.png">


Lets dig into more why it is seperating the background step in your case. Can you share the JSON produced by your tests?

Thanks, @newcat !

Reply:
I uploaded the [test setup](https://github.com/newcat/cucumber-jvm-background-problem-demo) including the [generated JSON](https://github.com/newcat/cucumber-jvm-background-problem-demo/blob/master/backgroundreport.json)